src/promptflow/promptflow/_sdk/data/docker/Dockerfile.jinja2

```diff
@@ -10,7 +10,9 @@ FROM docker.io/continuumio/miniconda3:latest
 
 WORKDIR /
 
-COPY ./flow /flow
+{% if env.python_requirements_txt %}
+COPY ./flow/{{env.python_requirements_txt}} /flow/{{env.python_requirements_txt}}
+{% end %}
 
 # create conda environment
 {% if env.conda_file %}
```

Comment:
it seems also need handle env.conda_file

Reply:
correct! fixed.